retrofit/gradle.properties

```diff
@@ -1,3 +1,5 @@
 POM_ARTIFACT_ID=retrofit
 POM_NAME=Retrofit
 POM_DESCRIPTION=A type-safe HTTP client for Android and Java.
+
+kotlin.stdlib.default.dependency=false
```

Comment:
Our version of Kotlin is old enough that the stdlib isn't added by default. This is a hedge against future upgrades.